app/src/test/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModelTest.kt

```diff
@@ -48,15 +51,18 @@ class TransactionDetailsViewModelTest {
     }
 
     @Test
+    @Ignore
```

Comment:
why is it ignored?

Reply:
Fixed in 6089571d0ad6e5fb42425ccec8725f310a0904c0